### ISSUE TITLE
How to predict new responses

### ISSUE THREAD

**Issue by continuesmile** [jkarimi91/chatbot#1]
Hi，thanks to your code，but there no code for predict，and what i want to konw is how to deal with the inputs containing unknown words  when predict new responses?
Thanks anyway.

**Comment by jkarimi91** [comment]
You are correct, there is no predict functionality yet (I am working on it); this repo is currently a work in progress. I will close this issue as soon as I have pushed the commit which includes this feature.

As far predicting on unknown words, typically, a special token meant to signify an unknown word is added to the vocabulary when encoding the input text as an int sequence. For example:
```python
input_str = 'hi jill how are you'
vocab = {'hi': 1, 'are': 2, 'i': 3, 'you': 4, 'fine': 5, 'am': 6, 'how': 7}
unknown_token = 0
input_seq = [vocab[word] if word in vocab else unknown_token for word in input_str.split()]
print input_seq
# [1, 0, 7, 2, 4]
```
